types/expression-evaluator/tslint.json

```diff
@@ -0,0 +1,6 @@
+{ 
+    "extends": "dtslint/dt.json",
```

Comment:
New definition packages cannot have anything in .json file other than extends clause. Please remove the rules from there and fix the errors.